packages/web/utils/rankHelpers.ts

```diff
@@ -1,31 +1,24 @@
-import { PlayerRank_Enum } from 'graphql/autogen/types';
-
-export const RANKS = [
-  PlayerRank_Enum.Diamond,
-  PlayerRank_Enum.Platinum,
-  PlayerRank_Enum.Gold,
-  PlayerRank_Enum.Silver,
-  PlayerRank_Enum.Bronze,
-];
-
-export const PLAYERS_PER_RANK = [7, 7, 7, 14, 21];
-
-// A summation of PLAYERS_PER_RANK.
-// This is the first index for which players will NOT be ranked, e.g.
-// the 56th player will be Bronze, and the 57th player will not be ranked.
-export const RANKED_CAP: number = PLAYERS_PER_RANK.reduce((sum, rankCount) => {
-  return sum + rankCount;
-}, 0);
+// A summation of usersPerRank
+// This is the first index for which users will NOT be ranked
+const computeRankCap = (usersPerRank: Array<number>) =>
+  usersPerRank.reduce((sum, rankCount) => {
+    return sum + rankCount;
+  }, 0);
 
 // Computes the rank for the given index. This would be the index corresponding
 // to all players ordered by total_xp DESC.
-export function computeRank(totalRankIndex: number): PlayerRank_Enum | null {
-  if (totalRankIndex >= RANKED_CAP) return null;
+export function computeRank(
```

Comment:
Yup, can you move this to the utils package and update the backend code to use it too?

Reply:
Done :)